pkg/config/validation/validation.go

```diff
@@ -1106,10 +1114,11 @@ var ValidateSidecar = registerValidateFunc("ValidateSidecar",
 					}
 					udsMap[bind] = struct{}{}
 				} else {
-					if _, found := portMap[egress.Port.Number]; found {
+					p := ipPortPair{ip: egress.Bind, port: egress.Port.Number}
```

Comment:
[nit] similarly to the `ingress` listener case, I suggest to document here why we're allowing the `bind` combinations like:

1. `""` and `127.0.0.1`